packages/dataviews/src/filters.tsx

```diff
@@ -37,17 +45,12 @@ const Filters = memo( function Filters( {
 			field: field.id,
 			name: field.header,
 			elements: field.elements,
-			singleSelection: operators.some( ( op ) =>
-				[ OPERATOR_IS, OPERATOR_IS_NOT ].includes( op )
+			singleSelection: operators.some(
+				( op ) => op === 'is' || op === 'isNot'
 			),
 			operators,
 			isVisible:
-				isPrimary ||
-				view.filters.some(
-					( f ) =>
-						f.field === field.id &&
-						ALL_OPERATORS.includes( f.operator )
```

Comment:
This is the other place I removed the operator validation. Again, if the operator is internal this should be fine. If they come from outside, it would be good to check them.

Reply:
They come from outside